tools/kompos/index.html

```diff
@@ -299,8 +311,8 @@
 
 <div id="code-views" class="card-columns"></div>
 
-<div class="card system-status" data-spy="affix">
-<div id="debugger" class="card-body">
+<div class="card system-status">
```

Comment:
Could you be more specific? What doesn't scroll? I haven't noticed any problems with scrolling.

It was introduced to fix the debugger on the top right, more than a year ago: https://github.com/smarr/SOMns/commit/ca94f9deb9d83fde1656b17db8b10150cc42a95f